README.md

```diff
@@ -1 +1,53 @@
-# To-do-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do list Project
+
+> This is my To Do list Microverse project!
+
+Created by:
+- Inaki Eskauriatza
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] [OPTIONAL] Please add the instruction to run the application locally to your readme in order to make it professional and also it will help non-technical people :smile: